boudicca.base/publisher-event-html/src/test/kotlin/base/boudicca/publisher/event/html/PublisherHtmlApplicationE2ETests.kt

```diff
@@ -0,0 +1,307 @@
+package base.boudicca.publisher.event.html
+
+import base.boudicca.api.search.QueryDTO
+import base.boudicca.api.search.SearchResultDTO
+import base.boudicca.model.Event
+import base.boudicca.publisher.event.html.fixture.E2ETestFixture
+import base.boudicca.publisher.event.html.service.SearchServiceCaller
+import base.boudicca.publisher.event.html.testdata.*
+import com.microsoft.playwright.assertions.PlaywrightAssertions.assertThat
+import com.microsoft.playwright.options.ElementState
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ArgumentsSource
+import org.mockito.kotlin.any
+import org.mockito.kotlin.whenever
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.boot.test.web.server.LocalServerPort
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import java.time.OffsetDateTime
+
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ExtendWith(SpringExtension::class)
+@Tag("playwright")
```

Comment:
maybe we can find a more generic term for that? "uitest" maybe?

Reply:
yes, you are right.